audit.py

```diff
@@ -36,16 +44,104 @@ def reverse_numeric_sort_order(number: int):
 # token=TOKEN -- required
 
 
-class OutputFormat(str, Enum):
-    text = "text"
-    json = "json"
-    html = "html"
+class OutputFormat(StrEnum):
```

Comment:
Wouldn't `OutputFormat = Literal["text", "json", "html"]` be a simpler solution?

Reply:
Using enum values in code instead of literal strings is a best practice when the exact string value has meaning.